src/key/key.js

```diff
@@ -692,7 +692,8 @@ class Key {
 
       results.push(...signatures.map(
         signature => ({
-          userID: user.userID.userID,
+          userID: user.userID ? user.userID.userID : null,
```

Comment:
I think better to like below?
```ts
userID: user.userID?.userID,
```

Reply:
Sure :+1: I'll convert it into a suggestion and apply it here :upside_down_face:
```suggestion
          userID: user.userID?.userID,
```

Reply:
Actually, I'll revert this as the Node.js tests complain

Reply:
Do you mean you want/have to support Node.js versions that are below v14?

Reply:
The CI tests on Node.js 14+ also failed, perhaps because of esm.js. Not sure, we can look into that separately later if we want to use this syntax

Reply:
Ok, Got it
Optional chaining is supported from Node.js v14 though.